src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -176,7 +182,35 @@ protected void updateItem(final TableRow item, final boolean empty) {
       this.workTableTreeView.getColumns().add(timeRangeColumn);
 
       final TreeTableColumn<TableRow, String> timeSumColumn = new TreeTableColumn<>("Duration");
-      timeSumColumn.setCellValueFactory(new TreeItemPropertyValueFactory<TableRow, String>("timeSum"));
+      timeSumColumn.setCellFactory(new Callback<TreeTableColumn<TableRow, String>, TreeTableCell<TableRow, String>>() {
+         @Override
+         public TreeTableCell<TableRow, String> call(TreeTableColumn<TableRow, String> tableRowStringTreeTableColumn) {
+
+            return new TreeTableCell<TableRow,String>(){
+
+               @Override
+               protected void updateItem(String s, boolean b) {
+                  super.updateItem(s, b);
+                  Label l1 = new Label(s);
```

Comment:
why create a label with a possibly null value? and later check 'l1.getText()!=null'. should make sure that s is not null and only then create a label with it

Reply:
Added your suggestion